src/profile-logic/tracks.js

```diff
@@ -248,14 +250,22 @@ export function computeLocalTracksByPid(
   const { counters } = profile;
   if (counters) {
     for (let counterIndex = 0; counterIndex < counters.length; counterIndex++) {
-      const { pid, category } = counters[counterIndex];
-      if (category === 'Memory') {
+      const { pid, category, sampleGroups } = counters[counterIndex];
+      if (category === 'Memory' || category === 'power') {
```

Comment:
(not for this patch) 
It's surprising to me that one has a capital "M" but the other has a lowercase "p". It feels error-prone. I would rather have only lowercase, but `Memory` is here for a long time now, so changing it would need an ugprader...